app/server/models/utils.py

```diff
@@ -98,9 +98,10 @@ def no_expire():
 def before_compile(query):
```

Comment:
SoftDelete mixin means that it uses the same code as `show_all` which builds up a query before every database query.